docs/authorization.md

```diff
@@ -230,6 +230,41 @@ i.e. a claimed resource is bound to the same maximal permission policy. Only the
 
 TBD: Example
 
+## Consumer Permission Claims Authorizer
```

Comment:
define what a consumer and provider is

Reply:
as discussed below, will rename it to `Restrict Permission Claims Authorizer`